springfield/base/templates/404.html

```diff
@@ -15,7 +15,7 @@
 {% endblock %}
 
 {# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
-{% block canonical_urls %}{% endblock %}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
```

Comment:
Is indexing of error pages a concern?

(That said, should this canonical block be shared right from the parent `base-error` template?)